types/jasmine/index.d.ts

```diff
@@ -436,58 +483,66 @@ declare namespace jasmine {
 
     interface Env {
         addReporter(reporter: CustomReporter): void;
-
-        execute(): void;
-        describe(description: string, specDefinitions: () => void): Suite;
-        // ddescribe(description: string, specDefinitions: () => void): Suite; Not a part of jasmine. Angular team adds these
-        beforeEach(beforeEachFunction: ImplementationCallback, timeout?: number): void;
-        beforeAll(beforeAllFunction: ImplementationCallback, timeout?: number): void;
-        afterEach(afterEachFunction: ImplementationCallback, timeout?: number): void;
-        afterAll(afterAllFunction: ImplementationCallback, timeout?: number): void;
-        xdescribe(desc: string, specDefinitions: () => void): XSuite;
-        it(description: string, func: () => void): Spec;
+        allowRespy(allow: boolean): void;
+        clearReporters(): void;
+        configuration(): EnvConfiguration;
+        configure(configuration: EnvConfiguration): void;
+        execute(runnablesToRun?: Suite[], onComplete?: Func): void;
+        /**
+         * @deprecated Use hideDisabled option in {@link jasmine.Env.configure} instead.
+         */
+        hideDisabled(value: boolean): void;
+        /**
+         * @deprecated Check hideDisabled option in {@link jasmine.Env.configuration} instead.
+         */
+        hidingDisabled(): boolean;
         // iit(description: string, func: () => void): Spec; Not a part of jasmine. Angular team adds these
-        xit(desc: string, func: () => void): XSpec;
-        addCustomEqualityTester(equalityTester: CustomEqualityTester): void;
-        addMatchers(matchers: CustomMatcherFactories): void;
-        specFilter(spec: Spec): boolean;
+        provideFallbackReporter(reporter: CustomReporter): void;
         /**
-         * @deprecated Use oneFailurePerSpec option in {@link jasmine.Env.configure} instead.
+         * @deprecated Use random option in {@link jasmine.Env.configure} instead.
          */
-        throwOnExpectationFailure(value: boolean): void;
+        randomTests(): boolean;
         /**
-         * @deprecated Use failFast option in {@link jasmine.Env.configure} instead.
+         * @deprecated Check random option in {@link jasmine.Env.configuration} instead.
```

Comment:
Use, not check

This is a setter not a getter.

Reply:
I believe the JSDoc should be identical to L502?

Reply:
they were switched